cli/src/main/java/bisq/cli/request/TradesServiceRequest.java

```diff
@@ -72,6 +78,22 @@ public TradeInfo getTrade(String tradeId) {
         return grpcStubs.tradesService.getTrade(request).getTrade();
     }
 
+    public List<TradeInfo> getOpenTrades() {
+        var request = GetTradesRequest.newBuilder()
+                .build();
+        return grpcStubs.tradesService.getTrades(request).getTradesList();
+    }
+
+    public List<TradeInfo> getTradeHistory(GetTradesRequest.Category category) {
+        if (!category.equals(CLOSED) && !category.equals(FAILED))
```

Comment:
1. Why not two methods: `getClosedTrades` and `getFailedTrades` or single method that handles all 3 categories?
2. Do we need category validation at this level?

Reply:
>> Why not two methods: getClosedTrades and getFailedTrades or single method that handles all 3 categories?
Every answer I can think of is splitting hairs, just like this question. ;-)

>> Do we need category validation at this level?
I think so.  It may seem redundant because the CLI validates the category param, but this is here for bots.